src/scss/_main.scss

```diff
@@ -644,39 +656,39 @@ nav {
   content: ' ';
 }
 
-.build-history .recent-build .-info {
-  padding: 0.2em 0.6em;
-
-  font-size: 14px;
-}
-
-.build-history .recent-build .-line {
+.recent-build-tooltip .line {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
 
-.build-history .-line.-header {
+.recent-build-tooltip .header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
 
-.build-history .-label {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-}
-
-.build-history .recent-build .-number {
+.recent-build-tooltip .number {
   margin-right: 0.5em;
 
   &:before {
     content: '#';
   }
 }
 
-.build-history .recent-build .-event {
+.recent-build-tooltip .info {
+  padding: 0.2em 0.6em;
+
+  font-size: 14px;
```

Comment:
should refrain from using fixed font size.

i'm planning to clean that up in a few places. in this case `0.9rem` might do the trick.